test/simulation/lntest/node.go

```diff
@@ -76,6 +82,15 @@ var (
 func generateListeningPorts() (int, int, int) {
 	var p2p, rpc, rest int
 	if numActiveNodes == 0 {
+	    freePorts, err := getFreePorts(3)
+		if err != nil {
+		    fmt.Println(fmt.Errorf("could not find free ports %v", err))
+		} else{
+            defaultNodePort = freePorts[0]
```

Comment:
No need to assign it to the default ports IMO. Just assign it to `p2p, rpc, rest` respectively.

Reply:
Then next node port's will get `19555 + (3 * numActiveNodes)` , I assigned it to the default ports just to make next node port finding dynamic too, am I wrong ?

Reply:
But the next node's port then won't be dynamic ? Am I wrong ?

Reply:
or should I use getFreePorts function whenever generate function is called ?

Reply:
https://github.com/ExchangeUnion/xud/pull/1182

Reply:
then I assume we can close this pR

Reply:
it's totally okay with me :)